test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -153,9 +154,10 @@ class ExtAuthzGrpcIntegrationTest
 
       *bootstrap.mutable_node()->mutable_metadata() = metadata;
 
-      envoy::config::listener::v3::Filter ext_authz_filter;
```

Comment:
These were the wrong type, but happened to work because the fields used had the same names and we do a proto->json conversion.